test/extended/networking/livemigration.go

```diff
@@ -70,7 +72,7 @@ var _ = Describe("[sig-network][OCPFeatureGate:PersistentIPsForVirtualization][F
 			DescribeTableSubtree("created using",
 				func(createNetworkFn func(netConfig networkAttachmentConfigParams) networkAttachmentConfig) {
 
-					DescribeTable("[Suite:openshift/network/virtualization] should keep ip", func(netConfig networkAttachmentConfigParams, vmResource string, opCmd func(cli *kubevirt.Client, vmNamespace, vmName string)) {
+					DescribeTable("[Suite:openshift/network/virtualization] should keep ip", func(netConfig networkAttachmentConfigParams, workloadConfig *workloadNetworkConfig, vmResource string, opCmd func(cli *kubevirt.Client, vmNamespace, vmName string)) {
```

Comment:
couldn't this be variadic input ? i.e. an optional attribute as the last item on the input parameter list ? That way, you'd avoid this ugly nil you sometimes pass around, plus the pointer.